packages/dds/map/src/map.ts

```diff
@@ -164,6 +164,8 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
      * Get an iterator over the entries in this map.
      * @returns The iterator
      */
+    // TODO: Use `unknown` instead (breaking change).
```

Comment:
With the suggestion above about moving the generic to the interface, most of these can become T instead of unknown I think.

Reply:
I'll make this change on next in the follow-up PR.